examples/src/main/kotlin/com/xebia/functional/xef/conversation/conversations/Animal.kt

```diff
@@ -34,13 +39,15 @@ suspend fun main() {
       val animal: Animal =
         AI(
           Prompt(model) { +user("A unique animal species.") }
-            .copy(configuration = configNoneFromConversation)
+            .copy(configuration = configNoneFromConversation),
+          conversation = this@Conversation
```

Comment:
Now we need to add the `conversation` in every call to OpenAI, otherwise we would be creating a new `conversationId` in each call